Open Judge System/Web/OJS.Web/Areas/Administration/Views/Contests/ChangeActiveParticipantsEndTime.cshtml

```diff
@@ -109,86 +81,121 @@
     <script>
         ($(function () {
             var timeInput = $('#TimeInMinutes');
+            var addButtonSelector = '#add-time-for-participants-btn';
+            var subtractButtonSelector = '#subtract-time-for-participants-btn';
+            var buttons = [addButtonSelector, subtractButtonSelector];
+
+            showTabWithError();
             changeAddSubractButton(timeInput.val());
 
-            timeInput.on('input',
-                function () {
-                    changeAddSubractButton(timeInput.val());
-                });
+            timeInput.on('input', function () {
+                changeAddSubractButton(timeInput.val());
+            });
+
+            $(buttons.join()).click(function () {
```

Comment:
Best to make it `.join(' ')`, i.e. put explicit whitespace as parameter. Some browsers, the older ones,  jon without a whitespace.